.github/workflows/pull_request_completed.yml

```diff
@@ -18,11 +18,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - name: Download and Extract Artifacts
-      uses: dawidd6/action-download-artifact@e7466d1a7587ed14867642c2ca74b5bcc1e19a2d # v6
+      uses: actions/download-artifact@v4
```

Comment:
The 2 are not quite the same. The former allows to `download this artifact in another workflow that is run after the first one.`.  Do we need that feature?

Reply:
Yes we do.

Reply:
`download-artifact@v4`  should now [support](https://github.com/actions/download-artifact/tree/v4?tab=readme-ov-file#download-artifacts-from-other-workflow-runs-or-repositories) this as well

Reply:
ooh good to know!